src/triage/cli.py

```diff
@@ -383,29 +383,37 @@ def __call__(self, args):
 class Db(Command):
     """Manage experiment database"""
 
-    @cmdmethod
+    @cmdmethod("revision", help="database schema revision to upgrade to (see triage db history)")
     def upgrade(self, args):
         """Upgrade triage results database"""
-        upgrade_db(dburl=self.root.db_url)
+        upgrade_db(revision=args.revision, dburl=self.root.db_url)
```

Comment:
I'm guessing the alembic upgrade command is OK with `revision=None` (and interprets this as `"head"`)?

Reply:
On second thought, won't this require that you specify the revision?

I think if you'd like it to be a regular old option, like: `triage db upgrade [-r REVISION]`:

    @cmdmethod('-r', '--revision', default='head')

Or, if you want it to take a single, but optional _argument_: `triage db upgrade [REVISION]`:

    @cmdmethod('revision', nargs='?', default='head')

(In both I've assumed that the default is required; though, perhaps it can/should fall back to `None`.)

I think the latter would probably make the most sense, (mirroring alembic).